ballerina/tests/pubsub_test.bal

```diff
@@ -41,10 +41,10 @@ function testGracefulShutdown() returns error? {
     stream<string, error?> subscribe = check pubsub.subscribe("topic");
     check pubsub.gracefulShutdown();
     record {|string value;|}|error? msg = subscribe.next();
-    if msg !is () {
-        test:assertFail(string `Expected "()", received ${(typeof msg).toString()}`);
+    test:assertTrue(msg is error);
+    if msg is error {
+        test:assertEquals(msg.message(), "Events must not be consumed from a closed pipe");
```

Comment:
Why we need to change the tests?

Reply:
We updated an error message in the pipe package, but the `pubsub` module hasn't yet been released with that pipe version, so this fix is needed.